Makefile

```diff
@@ -377,6 +377,7 @@ root-coverage: ## generate coverage profiles for unit tests that require root
 vendor:
 	@echo "$(WHALE) $@"
 	@go mod vendor
+	@go mod tidy
```

Comment:
It would be better to run `go mod tidy` before `go mod vendor`

Reply:
So the situation I ran into was that `go.sum` wasn't updated after running `make vendor`; would that case be resolved if we reverse? (curious)

Reply:
`go mod vendor` doesn't touch go.sum, so both can fix that. Just my thought that `go mod vendor` should be the last step.

Reply:
Just did some tests, if `go.mod` has some unused packages (which will be cleaned in `go mod tidy`), `go mod vendor` will copy that unused package info to `vendor/modules.txt`.

Reply:
Ah, thanks, makes sense. So my thinking was incorrect; I thought `go mod vendor` would determine the `go.sum` based on what was vendored.

I updated the PR and swapped the order

Reply:
Er.. meant "I thought `go mod tidy` (after `go mod vendor`) would determine the `go.sum` based on what was vendored)